oaknational/Oak-Web-Application
# feat(config): function which fetches secrets from google secret manager

## Description

- creates `fetchSecrets` function which will be used to fetch secrets at build time 
- uses authentication using service account form env

## Issue(s)

Relates to #35 

## How to test

1. Locally, add `fetchSecrets({ projectId: xxx, secretNames: [xxx,xxx] })` to the code and run `node fetch_secrets.js`
2. The script should fetch the secrets from Google Cloud secret manager and return them
